client/src/containers/App.js

```diff
@@ -17,6 +17,22 @@ import ScrollToTop from "components/ScrollToTop";
 import Modal from "components/Modal";
 import SocialShare from "components/SocialShare";
 import MethodologyPage from "./Methodology";
+import { withI18n } from "@lingui/react";
+
+const HomeLink = withI18n()((props) => {
+  const { i18n } = props;
+  return (
+    <Link
+      aria-label={i18n._(t`Who owns what in n y c?`)}
```

Comment:
Probably also out of scope but is there a reason we're not using "NYC" instead of "n y c"?  (I assume the reason for the spaces is so screen readers pronounce each letter?)

Reply:
That is exactly it. Let me add a comment to make that clearer.